core/src/main/java/nl/nn/adapterframework/configuration/digester/FrankDigesterRules.java

```diff
@@ -0,0 +1,139 @@
+/*
+   Copyright 2020 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.configuration.digester;
+
+import org.apache.commons.digester3.Digester;
+import org.apache.commons.digester3.ObjectCreationFactory;
+import org.apache.commons.digester3.Rule;
+import org.apache.commons.digester3.binder.LinkedRuleBuilder;
+import org.apache.commons.digester3.binder.RulesBinder;
+import org.apache.commons.digester3.binder.RulesModule;
+import org.apache.logging.log4j.Logger;
+
+import nl.nn.adapterframework.configuration.AttributeCheckingRule;
+import nl.nn.adapterframework.configuration.ConfigurationDigesterFactory;
+import nl.nn.adapterframework.configuration.GenericFactory;
+import nl.nn.adapterframework.configuration.JmsRealmsFactory;
+import nl.nn.adapterframework.configuration.ListenerFactory;
+import nl.nn.adapterframework.configuration.RecordHandlingFlowFactory;
+import nl.nn.adapterframework.util.LogUtil;
+
+public class FrankDigesterRules implements RulesModule {
+
+	private final Logger log = LogUtil.getLogger(this);
+	private GenericFactory factory = new GenericFactory();
+	private Rule attributeChecker = new AttributeCheckingRule();
+	private Digester digester;
+
+	public FrankDigesterRules(Digester digester) {
+		this.digester = digester;
+	}
+
+	@Override
+	public void configure(RulesBinder rulesBinder) {
```

Comment:
Ik zie hier nog niet de structuur van digester-rules.xml in terug, dus per pattern een 'create', 'set-properties' en 'register'